src/Swashbuckle.AspNetCore.SwaggerGen/Annotations/XmlCommentsOperationFilter.cs

```diff
@@ -98,7 +99,13 @@ private void ApplyResponsesXmlToResponses(IDictionary<string, Response> response
             {
                 // Check for a corresponding  API parameter (from ApiExplorer) that's property-bound?
                 var propertyParam = apiDescription.ParameterDescriptions
-                    .Where(p => p.ModelMetadata?.ContainerType != null && p.ModelMetadata?.PropertyName != null)
+                    .Where(p =>
+                    {
+                        var modelMetadata = p.ModelMetadata;
+                        return modelMetadata != null
+                            && modelMetadata.ContainerType != null
+                            && modelMetadata.PropertyName != null;
+                    })
```

Comment:
what's the advantage of this change vs just removing the second (redundant use of ?)? In other words, why not just replace:

    Where(p => p.ModelMetadata?.ContainerType != null && p.ModelMetadata?.PropertyName != null)

with

    Where(p => p.ModelMetadata?.ContainerType != null && p.ModelMetadata.PropertyName != null)

Is it just a style preference? If so, I still prefer the one-liner.